copy_unique/README.md

```diff
@@ -4,6 +4,25 @@ Copies unique files from one directory to another.
 
 Uniqueness is determined by hashing.
 
-This till is incredibly slow. [`rsync`] is a better tool.
+This tool is incredibly slow. [`rsync`][] is a better tool.
+
+## Installation
+
+It's suggested to use [`pipx`][]:
+
+```sh
+python -m pip install --user --upgrade pip setuptools wheel pipx
+python -m pipx ensurepath
+python -m pipx install 'git+https://github.com/mawillcockson/utilities#egg=copy_unique&subdirectory=copy_unique'
```

Comment:
If this tool is using [`poetry`](https://python-poetry.org/), would `pip` be able to install this project properly?

Reply:
According to [the `pip` documentation](https://github.com/pypa/pip/blob/6c09bdd7c0aff4bb48d9524b9e46d94d1830e9da/docs/html/reference/pip.rst#build-system-interface), the `pyproject.toml` will be referenced. For this project, that means `pip` will build it by using `poetry`.